keras_core/ops/image_test.py

```diff
@@ -101,3 +113,86 @@ def test_resize(self, method, antialias, data_format):
             ref_out = np.transpose(ref_out, (0, 3, 1, 2))
         self.assertEqual(tuple(out.shape), tuple(ref_out.shape))
         self.assertAllClose(ref_out, out, atol=0.3)
+
+    @parameterized.parameters(
+        [
+            ("bilinear", "constant", "channels_last"),
+            ("nearest", "constant", "channels_last"),
+            ("bilinear", "nearest", "channels_last"),
+            ("nearest", "nearest", "channels_last"),
+            # ("bilinear", "wrap", "channels_last"),  no wrap in torch
+            # ("nearest", "wrap", "channels_last"),  no wrap in torch
```

Comment:
You can use `if backend.backend() == "torch": pytest.skip("wrap fill_mode not support in torch")` so that at least it's tested when supported

Reply:
@fchollet 
When using `wrap`, the output in TF shows inconsistency compared to JAX and NumPy.
What make this situation awkward is that the test sometimes passes but at other times it fails.

As a solution, I have opted to skip the test to prevent CI failures.

Reply:
The mismatched elements is about 0.6%
```bash
Mismatched elements: 45 / 7500 (0.6%)  # one run for JAX
Mismatched elements: 48 / 7500 (0.64%)  # one run for NumPy
```